portfolio/src/main/java/com/google/sps/storage/DatastoreHelper.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.storage;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.IncompleteKey;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+
+import java.lang.reflect.*;
+
+public class DatastoreHelper <T>{
+    Datastore datastore;
+    KeyFactory keyFactory;
+    String kind;
+
+    public DatastoreHelper(String kind){
+        this.kind = kind;
+        datastore = DatastoreOptions.getDefaultInstance().getService();
+        keyFactory = datastore.newKeyFactory().setKind(kind);
+    }
+
+    public void put(T element){
+        Class<?> classType= element.getClass();
```

Comment:
It was definitely difficult to learn, but it works know. 

I am thinking of implementing a Blog feature in the future or have my projects, experiences and facts be implemented with the datastore database.

So it would help to avoid the implementation of methods specific class to entity conversion.